project/resources/js/Components/DataList.vue

```diff
@@ -1,22 +1,40 @@
+<template>
+    <div>
+        <slot name="options">
+        </slot>
+
+        <table class="col-12">
+            <thead>
+                <slot name="header" :orderBy="orderBy"></slot>
+            </thead>
+            <tbody>
+                <slot name="body" :fetchData="fetchData" :items="items"></slot>
+            </tbody>
+        </table>
+
+        <pagination
+          class="col-10"
+          :total-pages="totalPages"
+          :current-page="currentPage"
+          @fetchPrevPage="fetchPrevPage"
```

Comment:
`@fetchNextPage` and `@fetchPrevPage` may be omitted when using only `@changePage`